<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UI_Coordinate</class>
 <widget class="QDialog" name="UI_Coordinate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>240</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>240</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>坐标设置</string>
  </property>
  <property name="windowIcon">
   <iconset resource="mypic.qrc">
    <normaloff>:/icon/icons/324264.ico</normaloff>:/icon/icons/324264.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QCheckBox" name="auto_fit">
       <property name="text">
        <string>自适应</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="title">
        <string>坐标范围</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>10</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>10</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <property name="spacing">
             <number>20</number>
            </property>
            <item>
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>X轴</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <property name="spacing">
               <number>2</number>
              </property>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <property name="spacing">
                 <number>10</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>最小值</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="x_min">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>22</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <property name="spacing">
                 <number>10</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>最大值</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="x_max">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>22</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <property name="spacing">
             <number>20</number>
            </property>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Y轴</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <property name="spacing">
               <number>2</number>
              </property>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <property name="spacing">
                 <number>10</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>最小值</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="y_min">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>22</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <property name="spacing">
                 <number>10</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>最大值</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="y_max">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>22</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="xyset_ok">
         <property name="text">
          <string>确定</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="xyset_cancel">
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="mypic.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>xyset_cancel</sender>
   <signal>clicked()</signal>
   <receiver>UI_Coordinate</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xyset_ok</sender>
   <signal>clicked()</signal>
   <receiver>UI_Coordinate</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
